how-to/common/public/views/platform/of-info/of-info.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Displays a view with information about the OpenFin runtime.
+ */
+
+document.addEventListener('DOMContentLoaded', () => {
+	try {
+		init();
+	} catch (error) {
+		console.error(error);
+	}
+});
+
+/**
+ * Initialize the DOM elements.
+ */
+async function init() {
+	const rvmInfo = await fin.System.getRvmInfo();
+
+	const rvmVersionElement = document.querySelector('#rvmVersion');
```

Comment:
Can we add defensive coding for environments where the rvm isn't available?

Reply:
Added some defensive coding